### PR TITLE
add reductio v0.6.0 with npm auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/crossfilter/reductio
@PeterDaveHello  could u help me check this PR for #7434  ? thank you!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update) https://github.com/ddsky/unibox/issues/4
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Why don't you tag the main author in the commit but another one who has only 2 commits in the repo?

**Comment by maruilian11** [comment]
because the lib was belong to `esjewett/reductio` before, since it change to `crossfilter/reductio`, i thought tagging crossfilter member is better.

**Comment by PeterDaveHello** [comment]
Take a look at the commit history, esjewett is still the main author. Please update your commit.

**Comment by maruilian11** [comment]
can i tag both?

**Comment by PeterDaveHello** [comment]
Fine.

File: ajax/libs/reductio/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/crossfilter/reductio.git"
+  },
+  "keywords": [
+    "crossfilter"
+  ],
+  "author": "Ethan Jewett",
+  "license": "Apache-2.0",
+  "homepage": "https://github.com/crossfilter/reductio",
```
Comment (PeterDaveHello) [inline ajax/libs/reductio/package.json]: We don't need to set repo as homepage.

File: ajax/libs/reductio/0.5.4/reductio.js [hunk 2]
```diff
@@ -0,0 +1,1353 @@
+!function(e){if("object"==typeof exports&&"undefined"!=typeof module)module.exports=e();else if("function"==typeof define&&define.amd)define([],e);else{var f;"undefined"!=typeof window?f=window:"undefined"!=typeof global?f=global:"undefined"!=typeof self&&(f=self),f.reductio=e()}}(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
+(function (global){
+var reductio_build = require('./build.js');
+var reductio_accessors = require('./accessors.js');
+var reductio_parameters = require('./parameters.js');
+var reductio_postprocess = require('./postprocess');
```
Comment (PeterDaveHello) [inline ajax/libs/reductio/0.5.4/reductio.js]: We won't host this files, this may not work in browser, should confirm if it's need to host `reductio.js`

**Comment by PeterDaveHello** [comment]
This version is now old, should update to latest.

**Comment by maruilian11** [comment]
@PeterDaveHello 
by https://github.com/crossfilter/reductio/issues/39#issuecomment-222802673
>I think only the reductio.js and reductio.min.js files in the root directory are needed.

and i have updated the version to v0.6.0.

**Pull request merged**